src/PKSim.Core/Snapshots/Mappers/ProjectMapper.cs

```diff
@@ -16,55 +17,59 @@ public ProjectMapper(IExecutionContext executionContext)
          _executionContext = executionContext;
       }
 
-      public override SnapshotProject MapToSnapshot(ModelProject project)
+      public override async Task<SnapshotProject> MapToSnapshot(ModelProject project)
       {
-         return SnapshotFrom(project, snapshot =>
-         {
-            snapshot.Individuals = mapBuildingBlocksToSnapshots<Individual>(project.All<Model.Individual>());
-            snapshot.Compounds = mapBuildingBlocksToSnapshots<Compound>(project.All<Model.Compound>());
-            snapshot.Events = mapBuildingBlocksToSnapshots<Event>(project.All<PKSimEvent>());
-            snapshot.Formulations = mapBuildingBlocksToSnapshots<Formulation>(project.All<Model.Formulation>());
-            snapshot.Protocols = mapBuildingBlocksToSnapshots<Protocol>(project.All<Model.Protocol>());
-            snapshot.Populations = mapBuildingBlocksToSnapshots<Population>(project.All<Model.Population>());
-         });
+         var snapshot = await SnapshotFrom(project);
+         snapshot.Individuals = await mapBuildingBlocksToSnapshots<Individual>(project.All<Model.Individual>());
+         snapshot.Compounds = await mapBuildingBlocksToSnapshots<Compound>(project.All<Model.Compound>());
+         snapshot.Events = await mapBuildingBlocksToSnapshots<Event>(project.All<PKSimEvent>());
+         snapshot.Formulations = await mapBuildingBlocksToSnapshots<Formulation>(project.All<Model.Formulation>());
+         snapshot.Protocols = await mapBuildingBlocksToSnapshots<Protocol>(project.All<Model.Protocol>());
+         snapshot.Populations = await mapBuildingBlocksToSnapshots<Population>(project.All<Model.Population>());
```

Comment:
@rwmcintosh Observed data export is missing here.  can you add that when you do import?